portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="about.css">
+    <script src="truths_and_lies.js"></script>
+  </head>
+  <body>
+    <div class="about_content">
+    <div id="header">
+        <a href="index.html"> <img src=/images/profile.jpg /> </a>
+        <div class="left">
+            <a href="#"> <p> About </p> </a>
+            <a href="survey.html"> <p> Survey </p> </a>
+            <a href="contact.html"> <p> Contact </p> </a>
+        </div>
+    </div>
+    <div id="about">
+        <h3>Want to get to know me? Let's play Two Truths One Lie</h3>
+        <div id="truth_and_lies_game">
+            <label id="question_num">Press Start to Begin!</label>
+            <div id="questions">
+
+                <!-- These fields will be populated by the about.js file -->
+
+                <div class="truth_and_lies_question">
+                    <label>
+                        <input type="radio" name="chosen_answer" id="first_answer" value="0">
+                        <img src="/images/two_truth_one_lie_imgs/stars.jpg" id="first_img">
+                    </label>
+                    <p id="first_txt"></p>
```

Comment:
Hyphen instead of underscore here.

Also, I'd recommend being a bit more specific in this variable naming.  There is a lot of "text" on this page.  What we have here is some text that is either a true statement or a lie.  What's a common word for that?  Maybe "statement" or "claim"?
first-claim, second-claim, third-claim?

Reply:
Or maybe include the word "text" after all of that, since i just realized that you have also are identifying the images and might want to continue clearly distinguishing the <p> elements from the <img> elements.
So something like first-claim-text, second-claim-text, third-claim-text.

Reply:
Changed names